first_cycling_api/race/endpoints.py

```diff
@@ -83,16 +83,12 @@ def _parse_soup(self):
 		super()._parse_soup()
 		self._get_results_table()
 		self._get_sidebar_information()
-
+        
 	def _get_results_table(self):
-		results_table = self.soup.find('table', {'class': 'sortTabell tablesorter'})
-		if not results_table:
-			results_table = self.soup.find('table', {'class': 'sortTabell2 tablesorter'})
-		self.results_table = parse_table(results_table)
-
 		# Load all classification standings after stage
-		divs = self.soup.find_all('div', {'class': "tab-content dummy"})
+		divs = self.soup.find_all('div', {'class': "tab-content"}) #includes also tab-content results
 		self.standings = {div['id']: parse_table(div.table) for div in divs}
+		self.results_table = self.standings[divs[0]['id']] #first appearing is the result
```

Comment:
When I follow the first example in the README.md (for 2019 Amstel), this line is giving me the error below. Do you get this too?

```
File ~/FirstCyclingAPI/first_cycling_api/race/endpoints.py:91, in RaceEditionResults._get_results_table(self)
     89 divs = self.soup.find_all('div', {'class': "tab-content"}) #includes also tab-content results
     90 self.standings = {div['id']: parse_table(div.table) for div in divs}
---> 91 self.results_table = self.standings[divs[0]['id']]

IndexError: list index out of range
```

Reply:
Ok, will check. I should have left the fallback probably.